style.css

```diff
@@ -18,3 +28,64 @@
   background: linear-gradient(transparent, #FFD700, transparent);
   margin: 0 30px;
 }
+
+.feature-group {
+  background-color: #EEE;
+  padding: 40px;
+  position:relative;
+}
+
+.screen-width {
+  display: flex;
+  justify-content: center;
+  max-width: 1280px;
+  margin: 0 auto;
+  align-items: center;
+}
+
+.feature {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex-grow: 0;
+  margin: 0 50px;
+  max-width: 200px;
+}
+
+.black-font {
+  color: #111;
+  margin-bottom: 20px;
+}
+
+.grey-font {
+  color:#999;
+}
+
+.font {
+  display: flex;
+  font-weight: bold;
+  flex-wrap: wrap;
+  text-align: center;
+  }
+
+.feature-image{
+  align-items: baseline;
+  margin-bottom: 20px;
+}
+
+
+.feature-group::before {
+  background: url("images/triangle.png");
+  height: 12px;
+  content: "";
+  width: 100%;
+  top: -12px;
+  left: 0;
+  position: absolute;
+}
+
+@media screen and (max-width: 1024px) {
```

Comment:
you should remove the phone and the separator in the media query too.

Reply:
by remove you mean to be oriented as column instead of row?

Reply:
no, the phone and the separator should be hidden in responsive design.